proposals/low-level-struct-improvements.md

```diff
@@ -351,12 +367,25 @@ Span<int> ComplexScopedRefExample(scoped ref Span<int> span)
 
 <a name="rules-method-arguments-must-match"></a>
 
-The presence of `scoped` allows us to also refine the method arguments must match rule. It can now be reduced to 
+The presence of `ref` fields means the rules around method arguments must match need to be updated as a `ref` parameter can now be stored as a field in a `ref struct` argument to the method. Previously the rule only had to consider another `ref struct` being stored as a field. The impact of this is discussed in [the compat considerations](#compat-considerations). 
```

Comment:
>The presence of `ref` fields means the rules around method arguments must match need to be updated [](http://example.com/codeflow?start=0&length=98)

"The presence of `ref` fields means the rules around method arguments must match need to be updated ..."
This part of the sentence seems incomplete (I couldn't parse it).

Reply:
The docs uses the non `-` form for this.

Reply:
The other places with "method arguments must match" were not as hard to parse.
Consider: "... means the method arguments must match rule ..."